macosx/Controller.m

```diff
@@ -1269,8 +1269,14 @@ - (void) openURL: (NSString *) urlString
             else
                 urlString = [@"http://" stringByAppendingString: urlString];
         }
+        
+        NSURL * url = [NSURL URLWithString: urlString];
+        if (url == nil)
+        {
+            return;
```

Comment:
I'd log here for potential troubleshooting.

Reply:
Added logging.